docs/operations/operations.md

```diff
@@ -116,6 +116,23 @@ An example of the referenced secret containing the credentials for the GCP Cloud
 
 #### Permissions for GCP Cloud Storage
 
-Please make sure the service account associated with the provided credentials has the following IAM roles. 
+Please make sure the service account associated with the provided credentials has the following IAM roles.
 - [Storage Admin](https://cloud.google.com/storage/docs/access-control/iam-roles)
 
+### In-place vs Rolling-updates of Shoot Workers
+
+Changes to the `Shoot` worker-pools are applied in-place where possible. In case this is not possible a rolling update of the workers will be performed to apply the new configuration, as outlined in [the Gardener documentation](https://github.com/gardener/gardener/blob/master/docs/usage/shoot-operations/shoot_updates.md#in-place-vs-rolling-updates). The exact fields that trigger this behaviour depend on whether the feature gate `NewWorkerPoolHash` is enabled. If it is not enabled, only the fields mentioned in the [Gardener doc](https://github.com/gardener/gardener/blob/master/docs/usage/shoot-operations/shoot_updates.md#rolling-update-triggers) are used.
+If the feature gate _is_ enabled, instead of the complete provider config only the following fields are used:
+
+- `.spec.provider.workers[].dataVolumes[].name`
+- `.spec.provider.workers[].dataVolumes[].size`
+- `.spec.provider.workers[].dataVolumes[].type`
```

Comment:
This was missing from the actual change and created https://github.com/gardener/gardener-extension-provider-gcp/pull/979 to fix that